### PR TITLE
feat(VSInputOutput): Added input and output in vs format.

### PR DESCRIPTION
**Pull request by MelchiorSchuh**

### CODE REVIEW

File: bindings/python/src/CMakeLists.txt [hunk 1]
```diff
@@ -18,5 +18,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-add_subdirectory(mesh)
-add_subdirectory(model)
+add_subdirectory(mesh_geosciencesio)
```
Comment (BotellaA) [inline bindings/python/src/CMakeLists.txt:21]: mesh_geosciencesio or geosciencesio_mesh

File: src/geode/geosciences_io/mesh/common.cpp [hunk 2]
```diff
@@ -23,8 +23,10 @@
 
 #include <geode/geosciences_io/mesh/common.hpp>
 
-#include <geode/mesh/io/regular_grid_input.hpp>
-#include <geode/mesh/io/triangulated_surface_input.hpp>
+// #include <geode/mesh/io/point_set_input.hpp>
```
Comment (BotellaA) [inline src/geode/geosciences_io/mesh/common.cpp:26]: to remove?

**Review by BotellaA — APPROVED** [review]

File: bindings/python/src/CMakeLists.txt (hunk 1 above)
Comment (MelchiorSchuh) [reply bindings/python/src/CMakeLists.txt:21]: I put it at the end in order to have lib name + project name if lib name is not enough

**Pull request merged**